docs(readme): update and fix grammar (#2662)
kyranet/Skyra: README.md

## Translating Skyra <a href="https://translation.skyra.pw" target="_blank"><img src="https://support.crowdin.com/assets/logos/crowdin-core-logo.png" align="right" width="30%"></a>

We use **Crowdin** to translate Skyra's messages into different languages. If you'd like to help by contributing new

> translations (or improving existing ones), [**click here**](https://translation.skyra.pw). Thanks for any contributions!

Reviewer: ```suggestion
translations (or improving existing ones), [**click here**](https://translation.skyra.pw). Thank you in advance for any contributions!
```

## A note to aspiring developers who want to self-host Skyra

> The developer team does not support the idea of other self-hosted instances of Skyra. The team prides itself on

Reviewer: ```suggestion
The development team does not support the idea of other self-hosted instances of Skyra. The team prides itself on
```

providing the best experience and support for the end-users. As such, an offshoot or unaffiliated mirror of Skyra may
cause ill effects on the reputation and image of Skyra. If you wish to see new features implemented, please refer to the

> developing guidelines linked above.

Reviewer: ```suggestion
development guidelines linked above.
```
